docs/reference-docs/rug/rug-conventions.md

```diff
@@ -28,22 +26,23 @@ concrete examples.
 
 [travis-editors]: https://github.com/atomist-rugs/travis-editors
 
-### Rug Archive
+### Rug Projects
```

Comment:
It feels appropriate to just fix doc bugs whenever, but the question of whether it makes sense to just fix it here is a good one.